### PR TITLE
[Term Entry] Kotlin Math .sin()

### LINKED ISSUE #3629: [Term Entry] Kotlin Math .sin()
### Reason/inspiration (optional)

We would like a new term entry on the `.sin()` method for Kotlin: [Math](https://www.codecademy.com/resources/docs/kotlin/math-methods). The entry should go in a new file under `docs/content/kotlin/concepts/math-methods/terms/sin/sin.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```kotlin `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by rahul0x00**
### Description
Added Entry for Kotlin Math.sin() 
Closes #3629 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/math-methods/terms/sin/sin.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+Title: 'Math.sin()'
```
Comment (caupolicandiaz) [inline content/kotlin/concepts/math-methods/terms/sin/sin.md:2]:
```suggestion
Title: 'sin()'
```

Comment (caupolicandiaz) [reply content/kotlin/concepts/math-methods/terms/sin/sin.md:2]: This entry has many of the same issues as the other math submissions you've completed. Please look at the previously reviewed entries and make the required updates.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@rahul0x00 Please update, and tag when complete. Thanks

**Comment by rahul0x00** [comment]
@caupolicandiaz updated the entry

**Comment by caupolicandiaz** [comment]
> @caupolicandiaz updated the entry

This is not complete. Almost all the same edits made in the other kotlin math entries you made are applicable here. I did not individually highlight them, but you should be able to compare and make the changes.

**Comment by rahul0x00** [comment]
@caupolicandiaz Hi, can you review the changes?

**Review by caupolicandiaz — APPROVED** [review]
@rahul0x00 I made a few additional direct edits. This is ready for a second review.

**Review by SSwiniarski — APPROVED** [review]
@rahul0x00 Looks good.

**Pull request merged**